common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -676,6 +676,23 @@ def login_analytics(strategy, auth_entry, current_partial=None, *args, **kwargs)
         )
 
 
+def emit_login_event(strategy, auth_entry, current_partial=None, *args, **kwargs):
```

Comment:
Please move this function to our own caliper app.

Reply:
done